ImperatorToCK3/Data_Files/blankMod/output/localization/english/CONVERTER_cultures_l_english.yml

```diff
@@ -1,106 +1,105 @@
-﻿# ImperatorToCK3: Last updated CK3 patch 1.2.2
```

Comment:
You removed the version note here.

Reply:
Put it back and updated it; ditto in the other files